### DISCUSSION TITLE
Config not used when in -no-gui

### DISCUSSION THREAD

**Discussion by eskimos20** [cagnulein/qdomyos-zwift#2956] [Q&A] [answered]
Hi! 

I have this setup right now.

Linux Mint 22 Cinnemon (switching to Ubuntu Server hopefully)
Asus BT500
Tacx Neo2T

I have everything working in QT app with gui where I have set "Get Gears from Zwift" and everything is woking OK. So far so good.

I thought that all settings when using gui would translate to the same when using no-gui mode. But when I use no-gui mode the "Get Gears from Zwift" is not working. 

When using -no-gui mode is there a .config file located somewhere where I can hardcode that value so it will work in no-gui mode aswell? 

I found this https://github.com/cagnulein/qdomyos-zwift/blob/master/docs/30_usage.md but it does not have a "Get Gears from Zwift" option.

/Eskimos

**Comment by cagnulein** [comment, marked as answer]
it's this

`-zwift_play_emulator`

**Reply by eskimos20** [reply]
Mmm that does not seem to do it for me. Tried:

sudo ./qdomyos-zwift -no-gui -no-log -zwift-play-emulator

Wahoo Kickr 0000 33 and QZPI gets enabled and are accessable through the pairing screen, but cant find Zwift Click like I normally do.

**Reply by cagnulein** [reply]
the click must be paired to zwift only in this way and the zwift click setting must be disabled in qz

**Reply by eskimos20** [reply]
Yes that is the settings im using when using the gui.. and that works like below picture. But not when using the no-gui command like i described. 



![20241229_104243](https://github.com/user-attachments/assets/04b58635-7a16-4d13-baf5-99577030aaf7)

**Reply by cagnulein** [reply]
share a debug log, by default zwift click is disabled on qz nogui, so it shouldn't interfere with zwift

**Comment by eskimos20** [comment]
Here are the debug log for the following command:

sudo ./qdomyos-zwift -no-gui -zwift-play-emulator

[debug-sön_dec__29_10_57_57_2024.log](https://github.com/user-attachments/files/18269073/debug-son_dec__29_10_57_57_2024.log)

**Comment by cagnulein** [comment]
The problem is that the zwift play emulator setting is disabled. I added
this switch 2 months ago, is this build new? Otherwise this flag will be
ignored

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 29 dic 2024 alle 11:06 eskimos20 ***@***.***>
ha scritto:

**Reply by eskimos20** [reply]
Yes I pulled from git yesterday according to this, on linux from source:

https://github.com/cagnulein/qdomyos-zwift/blob/master/docs/10_Installation.md

Is that correct ?

**Comment by cagnulein** [comment]
You wrote it wrong

-zwift_play_emulator

Not 

-zwift-play-emulator

**Reply by eskimos20** [reply]
That worked!! Excellent. Thanks a lot for your help. Next time in Italy I will pay for Coffe :D But you have to ride outdoors then! :D

/Eskimos

**Comment by cagnulein** [comment]
I'm commuting 16 km by day, each day to work on my bike, so no problem :)
Remember to support the app on patreon!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 29 dic 2024 alle 11:25 eskimos20 ***@***.***>
ha scritto:
